### PR TITLE
Add repeated-test recipe

### PR DESCRIPTION
**Pull request by proinsias**
- [x] Tidy up `meta.yaml` when build succeeds

### CODE REVIEW

File: recipes/repeated-test/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six >=1.7
```
Comment (jakirkham) [inline recipes/repeated-test/meta.yaml]: Probably can drop this from build requirements.

File: recipes/repeated-test/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - repeated_test.tests
+
+  requires:
+    - unittest2  # [py26]
+
+about:
+  home: https://github.com/epsy/repeated_test
+  license: MIT
+  license_family: MIT
+  # license_file: LICENSE.txt # Waiting on new release
```
Comment (jakirkham) [inline recipes/repeated-test/meta.yaml]: Is this why it is WIP? If so, I'd rather just merge and save that for the feedstock. Though am not going to push it if you would rather wait. What do you think?

**Comment by jakirkham** [comment]
Any thoughts on this, @proinsias?

File: recipes/repeated-test/meta.yaml (hunk 1 above)
Comment (proinsias) [inline recipes/repeated-test/meta.yaml]: Checking... Will tidy up `meta.yaml` and update PR title once it's done.

File: recipes/repeated-test/meta.yaml (hunk 2 above)
Comment (jakirkham) [inline recipes/repeated-test/meta.yaml]: xref: https://github.com/epsy/repeated_test/pull/4

File: recipes/repeated-test/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+#    - six >=1.7
```
Comment (jakirkham) [inline recipes/repeated-test/meta.yaml]: This should be dropped, right?

File: recipes/repeated-test/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
+{% set name = "repeated_test" %}
+{% set version = "1.0" %}
+{% set sha256 = "73193f2bb360051ec78faeadc72b8b70240852e3f2746ff1da1526f4c03be33a" %}
```
Comment (jakirkham) [inline recipes/repeated-test/meta.yaml]: Please bump to `1.0.1` and update the checksum.

File: recipes/repeated-test/meta.yaml (hunk 2 above)
Comment (jakirkham) [inline recipes/repeated-test/meta.yaml]: Please uncomment it is included in `1.0.1`.

**Comment by proinsias** [comment]
@jakirkham – should be all set now.

**Comment by jakirkham** [comment]
Thanks @proinsias.

File: recipes/repeated-test/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,45 @@
+{% set name = "repeated_test" %}
```
Comment (jakirkham) [inline recipes/repeated-test/meta.yaml]: Sorry, missed this earlier. Appears the directory and package name don't match. Should it be with a `-` or an `_`?

**Comment by jakirkham** [comment]
This recipe failed conversion for the reason above. Please add a new PR with this recipe and the above issue corrected.

**Comment by jakirkham** [comment]
Added issue ( https://github.com/conda-forge/staged-recipes/issues/1660 ) for resolving this issue and getting this addition in a feedstock.

File: recipes/repeated-test/meta.yaml (hunk 5 above)
Comment (jakirkham) [inline recipes/repeated-test/meta.yaml]: Another occurrence of issue ( https://github.com/conda-forge/staged-recipes/issues/1073 ).

**Pull request merged**